Mathlib/Order/GaloisConnection.lean

```diff
@@ -204,6 +204,9 @@ theorem u_eq_top {l : α → β} {u : β → α} (gc : GaloisConnection l u) {x}
 theorem u_top [OrderTop β] {l : α → β} {u : β → α} (gc : GaloisConnection l u) : u ⊤ = ⊤ :=
   gc.u_eq_top.2 le_top
 
+theorem l_u_top {l : α → β} {u : β → α} (gc : GaloisConnection l u) : u (l ⊤) = ⊤ :=
```

Comment:
Can you add the corresponding theorem about `bot` in the `OrderBot` section below?

Reply:
Yeah, I added it.

I know only enough about Galois Connections to be dangerous.  Can you double check the names of the functions/theorems?  I would suspect that the l_u_top really should be u_l_top.  Similarly, I suspect that the function/theorem I just wrote (u_l_bot) really should be (l_u_bot).  The first name was suggested to me in a [zulip chat.](https://leanprover.zulipchat.com/#narrow/channel/287929-mathlib4/topic/preimage_of_image_univ.20missing.20in.20Mathlib.3F)

Reply:
I agree, these names sound backwards to me

Reply:
I have written the preimage_image_empty theorem.  I also added a image_preimage_empty theorem for which I used a tactic proof.

The missing theorem (image_preimage_univ) isn't true.